apps/ledger-live-desktop/tests/specs/speculos/swap.spec.ts

```diff
@@ -182,6 +181,8 @@ const swaps = [
 ];
 
 for (const { swap, xrayTicket } of swaps) {
+  test.setTimeout(process.env.CI ? 600000 : 1200000);
```

Comment:
to avoid adding this in every swap test just set it at the beginning of the file, for example like this:

```
test.setTimeout(process.env.CI ? 600000 : 1200000);
const app: AppInfos = AppInfos.EXCHANGE;
```